src/tinypeel/tinypeel.py

```diff
@@ -213,19 +213,166 @@ def generateSingleLocusSegregation(peelingInfo, pedigree, args):
+        type=str,
+        nargs="*",
+        help="A file in plink (binary) format. Only stable on Linux.",
+    )
+    parse_dictionary["genotypes"] = lambda parser: parser.add_argument(
+        "-geno_file",
+        default=None,
+        required=False,
+        type=str,
+        nargs="*",
+        help="A file in AlphaGenes format.",
+    )
+    parse_dictionary["reference"] = lambda parser: parser.add_argument(
+        "-reference",
+        default=None,
+        required=False,
+        type=str,
+        nargs="*",
+        help="A haplotype reference panel in AlphaGenes format.",
+    )
+    parse_dictionary["seqfile"] = lambda parser: parser.add_argument(
+        "-seq_file",
+        default=None,
+        required=False,
+        type=str,
+        nargs="*",
+        help="A sequence data file.",
+    )
+    parse_dictionary["pedigree"] = lambda parser: parser.add_argument(
+        "-ped_file",
+        default=None,
+        required=False,
+        type=str,
+        nargs="*",
+        help="A pedigree file in AlphaGenes format.",
+    )
+    parse_dictionary["phasefile"] = lambda parser: parser.add_argument(
+        "-hap_file",
+        default=None,
+        required=False,
+        type=str,
+        nargs="*",
+        help="A haplotype file in AlphaGenes format.",
+    )
+    parse_dictionary["startsnp"] = lambda parser: parser.add_argument(
+        "-start_snp",
+        default=None,
+        required=False,
+        type=int,
+        help="The first marker to consider. The first marker in the file is marker '1'. Default: 1.",
+    )
+    parse_dictionary["stopsnp"] = lambda parser: parser.add_argument(
+        "-stop_snp",
+        default=None,
+        required=False,
+        type=int,
+        help="The last marker to consider. Default: all markers considered.",
+    )
+    parse_dictionary["seed"] = lambda parser: parser.add_argument(
+        "-seed",
+        default=None,
+        required=False,
+        type=int,
+        help="A random seed to use for debugging.",
+    )
+
+    return parse_dictionary
+
+
+def get_output_options():
+    parse_dictionary = dict()
+
+    parse_dictionary["writekey"] = lambda parser: parser.add_argument(
+        "-out_id_order",
+        default="id",
+        required=False,
+        type=str,
+        help='Determines the order in which individuals are ordered in the output file based on their order in the corresponding input file. Animals not in the input file are placed at the end of the file and sorted in alphanumeric order. These animals can be surpressed with the "-onlykeyed" option. Options: id, pedigree, genotypes, sequence, segregation. Defualt: id.',
+    )
+    parse_dictionary["onlykeyed"] = lambda parser: parser.add_argument(
+        "-out_id_only",
+        action="store_true",
+        required=False,
+        help='Flag to surpress the animals who are not present in the file used with -outputkey. Also surpresses "dummy" animals.',
```

Comment:
Change `outputkey` to `???`

Reply:
I'm not sure if you pointed at `writekey`, it has been changed to `out_id_order`.

Reply:
It should be "-out_id_order". In the documents, it is right.